examples/rust/get_started/examples/09-secure-channel-over-many-transport-hops-initiator.rs

```diff
@@ -11,26 +11,18 @@ async fn main(mut ctx: Context) -> Result<()> {
     // Create a TCP connection.
     tcp.connect("127.0.0.1:3000").await?;
 
-    let vault = Vault::create(&ctx)?;
+    let mut initiator = LocalEntity::create(&ctx, "initiator").await?;
+    let middle: Address = (TCP, "127.0.0.1:3000").into();
+    let responder: Address = (TCP, "127.0.0.1:4000").into();
+    let route: Route = vec![middle, responder, "secure_channel_listener".into()].into();
```

Comment:
Love that syntax. Would be even better if we could do that with slice (not sure it's possible)